lessons/Projects/project-video-game/langs/en-us/index.adoc

```diff
@@ -1,102 +1,145 @@
 
+=== Launch
 
-[cols=".^1a, .^1a, .^1a" options="header", stripes="none"]
+There are six phases of video game creation. Each phase (except for the first and the last) requires that you teach:
+
+- up to four lessons during which students learn relevant coding skills and algebra concepts
+- one lesson where students use what they've learned to update the code of their @starter-file{game, "Game Starter File"}
+
+In the first two phases (below), students conceptualize their game and then add images to their @starter-file{game, "Game Starter File"}. The table below highlights (1) the lessons you will teach to reinforce relevant algebra and coding concept, and (2) the lessons where students will update their personal video games.
+
+[cols="^.^1,<.^3,<.^6,<.^3", stripes="none", options="header"]
 |===
-| *Game Section*	| *Lessons* | *Workbook Pages*
-| *0{empty}.  Game title*
+| #
+| Phase
+| First, Learn Code and Algebra:
+| Then, Update the Game:
+
+| 1
+| Conceptualization
+| _NA_
 | @lesson-link{coordinates}
 
-| @printable-exercise{coordinates/pages/BrainstormGame.adoc}
+| 2
+| Game Imagery
+| @lesson-link{order-of-operations}, @lesson-link{simple-data-types}, @lesson-link{contracts}, @lesson-link{defining-values}
+| @lesson-link{making-game-images}
 
-| *1{empty}. Making the Danger and the Target Move*
-| @lesson-link{functions-for-character-animation}
+|===
+
+By the end of Phase 1, each student will have created a "screenshot" of their video game by inserting images that they find on the web into a @online-exercise{https://docs.google.com/drawings/d/1xXNWcbJ7KOgTClFQeHq-tIy0xmeZds9N3OktvEBaO4c/copy, Google Draw template}. Along the way, students will think about horizontal and vertical position as represented by ordered pairs.
+
+In Phase 2, students acquire the skillset needed to actually add game images to section 0 of their @starter-file{game, "Game Starter File"}. The programming concepts that students learn here form a foundation upon which to build, and will prove essential as you work your way through the remaining lessons. Students actually edit and update their games during @lesson-link{making-game-images}.
 
-| @ifproglang{pyret}{@printable-exercise{functions-for-character-animation/pages/character-animation-dr.adoc}} @ifproglang{wescheme}{@printable-exercise{functions-for-character-animation-wescheme/pages/character-animation-dr.adoc}}
+During @lesson-link{making-game-images}, we recommend printing and distributing the student-facing @opt-printable-exercise{video-game-rubric.adoc, "rubric"}. This document will highlightfor students the scope of the project and your expectations. Direct your students' attention to Rubric Row 1 ("Images") to clarify your requirements for this component of their video game. (We encourage you, of course, to edit the rubric to meet your classroom needs!)
```

Comment:
"X will highlight for students Y" -- still more complex than it needs to be. I'll push revised wording. Let me know if you're ok with it.

Reply:
Also, why is the rubric an opt-printable instead of a handout? Is it because using it is merely a _recommendation_ rather than a demand? If so, do we do that for rubrics in other projects?